docs/api/en/math/Triangle.html

```diff
@@ -126,6 +126,16 @@ <h3>[method:Triangle set]( [param:Vector3 a], [param:Vector3 b], [param:Vector3
 		Please note that this method only copies the values from the given objects.
 		</p>
 
+		<h3>[method:Triangle setFromAttributeAndIndices]( [param:BufferAttribute attribute], [param:Integer i0], [param:Integer i1], [param:Integer i2] ) [param:Triangle this]</h3>
+		<p>
+		attribute - [page:BufferAttribute] of vertex data <br />
+		i0 - [page:Integer] index <br />
+		i1 - [page:Integer] index <br />
+		i2 - [page:Integer] index<br /><br />
+
+		Sets the triangle's vectors to the vectors in the buffer attribute.
```

Comment:
Something like

    Sets the triangle's vertices from the buffer attribute vertex data.